# DO NOT MODIFY -- direct copy of kuri message
uint16[] adc_channels
float32 v_dock
float32 v_batt
float32 i_batt
float32 t_batt
bool dock_present
bool is_charging
bool power_button_pressed
BatteryCapacity battery
